src/metabase/core.clj

```diff
@@ -2,9 +2,7 @@
 (ns metabase.core
   (:gen-class)
   (:require [cheshire.core :as json]
-            [clojure
-             [pprint :as pprint]
-             [string :as s]]
+            [clojure.pprint :as pprint]
             [clojure.tools.logging :as log]
             environ.core
```

Comment:
I didn't see any references in core to `env`. Can we remove this?

Reply:
done